packages/m/mosh/package.yml

```diff
@@ -13,6 +13,8 @@ builddeps  :
     - pkgconfig(bash-completion)
     - pkgconfig(protobuf)
 setup      : |
+    # protobuf 23 requires C++17
+    sed -i 's|\[11\]|\[17\]|' -i $workdir/configure.ac
```

Comment:
I would suggest using a patch file instead of `sed`.

Reply:
How would I easily do this?